tests/integration/dart/test/credential_issuance_client_test.dart

```diff
@@ -93,49 +85,57 @@ void main() {
           // ..credentialSupported = ListBuilder(credentialSupported)
           //
           ;
-      final createdConfig = (await configurationApi.createIssuanceConfig(
-              createIssuanceConfigInput: configInputBuilder.build()))
-          .data;
-      expect(createdConfig!.id, isNotEmpty);
-
-      // get config
-      final configDetails = (await configurationApi.getIssuanceConfigById(
-              configurationId: createdConfig.id!))
-          .data;
-      print(configDetails);
-      expect(configDetails, isNotNull);
-      expect(configDetails!.id, equals(createdConfig.id));
-      expect(configDetails.issuerWalletId, equals(walletId));
-      expect(configDetails.name, equals(name));
-      expect(configDetails.description, equals(description));
-      expect(configDetails.format.toString(), equals(format.toString()));
-      expect(configDetails.credentialOfferDuration,
-          equals(credentialOfferDuration));
-      expect(configDetails.issuerUri, isNotEmpty);
-      expect(configDetails.issuerDid, isNotEmpty);
-      // expect(configDetails.credentialSupported, equals(credentialSupported));
 
-      // list config
+      // NOTE: only 1 issuance config can exist. Create a new test configuration
+      //       if there is no other configuration present.
       var configs =
           (await configurationApi.getIssuanceConfigList()).data!.configurations;
-      expect(configs, isNotNull);
-      expect(configs.length, equals(1));
-      expect(configs.first.id, equals(createdConfig.id));
-      expect(configs.first.issuerWalletId, equals(walletId));
-      expect(configs.first.name, equals(name));
-      expect(configs.first.format.toString(), equals(format.toString()));
-      expect(configs.first.credentialOfferDuration,
-          equals(credentialOfferDuration));
-      expect(configs.first.issuerUri, isNotEmpty);
-      expect(configs.first.issuerDid, isNotEmpty);
 
-      // delete config
-      await configurationApi.deleteIssuanceConfigById(
-          configurationId: createdConfig.id!);
-      configs =
-          (await configurationApi.getIssuanceConfigList()).data!.configurations;
-      expect(configs, isNotNull);
-      expect(configs.length, equals(0));
+      if (configs.length == 0) {
```

Comment:
Naive question, when can this length equal 0?
I'm writing this as there is no assertion when it's equal zero, thus this test might just pass, and potentially be a false positive that tests nothing (no ?).

Reply:
> Naive question, when can this length equal 0?
when you have no issuance configs. Since we can have only 1 issuance config, we don't want accidentally delete it (as it was setup before)